plugins/pencilblue/controllers/actions/admin/sites/new_site.js

```diff
@@ -48,7 +48,7 @@ module.exports = function NewSiteActionModule(pb) {
             var selectedLocale = self.body.selectedLocales[i];
             self.body.supportedLocales[selectedLocale] = true;
         }
-        self.body.selectedLocales = undefined;
+        delete self.body.selectedLocales;
```

Comment:
An oversite in original push for the locale submission. The original would include a null 'selectedLocales' entry to the db for the site object. This should clean that up for sites going forward, which raises another question:

For existing 'corrupted' sites with null 'site' and 'selectedLocale' entries, is it worth scrubbing those clean inside edit_site?